app/components/slack-context-menu.js

```diff
@@ -138,7 +149,7 @@
 	}
 
 	module.exports = {
-		handleRightClick: function (window, target, x, y) {
+		handleRightClick: function (window, target, x, y, teamUrl) {
```

Comment:
https://github.com/plaidchat/plaidchat/blob/master/app/components/slack-window.js#L243 looks like a standard prop! :P I was talking about https://medium.com/@learnreact/context-f932a9abab0e but they're not documented yet, hence the eventually.